sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -167,76 +157,6 @@ export namespace ConnectionContext {
       await connectionContext.cbsSession.close();
       // Close the management session to ensure all the event handlers are released.
       await connectionContext.managementSession!.close();
-
-      // The connection should always be brought back up if the sdk did not call connection.close()
-      // and there was atleast one sender/receiver link on the connection before it went down.
-      logger.verbose("[%s] state: %O", connectionContext.connection.id, state);
```

Comment:
It might be good to still log something to verbose indicating that the disconnected event was fired on the connection for troubleshooting purposes. Right now we'll only log if an error was detected, but sometimes there isn't an error (I believe the amqp idle_time_out does not disconnect with an error).

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/6737/commits/954784aa98b22d05080e7304f50373c3a76baf2b